test/transaction.js

```diff
@@ -208,9 +205,15 @@ it('multiSign() for x-of-5 should be deterministic', function () {
 
   var deserializedTx = zencashjs.transaction.deserializeTx(serializedTx)
   expect(deserializedTx).to.deep.equal(tx0)
+
+  // make sure order in which the signatures are presented doesn't matter
+  var tx0_unsorted = zencashjs.transaction.applyMultiSignatures(txobj, 0, [sig2, sig1], redeemScript)
+  // Serialize the transaction
+  var serializedTx_unsorted = zencashjs.transaction.serializeTx(tx0_unsorted)
+  expect(serializedTx_unsorted).to.equal(serializedTx)
```

Comment:
Typically variables in Javascript are written in camelCase

Reply:
Yes, modified accordingly.